packages/core/addon/models/bard-request-v2/request.ts

```diff
@@ -235,8 +239,8 @@ export default class RequestFragment extends Fragment.extend(Validations) implem
    * Sets the table of the request
    */
   setTableByMetadata(table: TableMetadata) {
-    this.set('table', table.id);
-    this.set('dataSource', table.source);
+    set(this, 'table', table.id);
+    set(this, 'dataSource', table.source);
```

Comment:
What is change for?

Reply:
Ember eslint wanted me to mark this as a `@classic` class or use the utility version https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/classic-decorator-no-classic-methods.md